src/usng.ts

```diff
@@ -560,6 +560,19 @@ extend(Converter.prototype, {
     }
   },
 
+  LLtoUPSString: function(lat, lon) {    
+    if (!this.isInUPSSpace(lat)) {
+      throw new Error(`usng.js, LLtoUPSString, invalid input. lat: ${lat.toFixed(4)} lon: ${lon.toFixed(4)}`);
+    }
+
+    const upsObject = this.LLtoUPS(lat, lon)
+    const upsZoneLetter =  upsObject.northPole
+    ? (upsObject.easting < 2000000 ? 'Y' : 'Z')
+    : (upsObject.northing < 2000000 ? 'A' : 'B')
+
+    return `${upsZoneLetter} ${Math.round(upsObject.easting)}mE ${Math.round(upsObject.northing)}mN`
```

Comment:
✏️ Might be useful to have a parse function for reversing this, just for easier use of this:
```
parseUpsString: function(upsString) {
  const parts = upsString.split(' ')
  const zone = parts[0]
  const easting = parts[1].replace('mE', '')
  const northing = parts[2].replace('mN', '') 

  let northPole = true
  if (zone === 'A' || zone === 'B') 
      northPole = false
  else if (zone === 'Y' || zone === 'Z')
      northPole = true
  
  return {
    northPole: northPole,
    hemisphere: northPole ? "NORTHERN" : "SOUTHERN",
    northing: northing,
    easting: easting,
  }
}
```

Reply:
For posterity, we found `deserializeUPS(upsString)`